src/stars.css

```diff
@@ -0,0 +1,23 @@
+
+.card__stars {
```

Comment:
it should be separate BEM block `stars`
```suggestion
.stars {
```